src/node_platform.cc

```diff
@@ -16,50 +16,52 @@ using v8::Platform;
 using v8::Task;
 using v8::TracingController;
 
-static void BackgroundRunner(void* data) {
+namespace {
+
+static void WorkerThreadMain(void* data) {
   TRACE_EVENT_METADATA1("__metadata", "thread_name", "name",
                         "BackgroundTaskRunner");
```

Comment:
Do we have to change this? @nodejs/trace-events

Reply:
It makes sense to change this to match the concept name actually used by V8, that is, 'WorkerThread'. However, things going to get really confusing since we also have `worker_threads` from https://github.com/nodejs/node/pull/20876.

One of these concepts needs to be renamed for sanity's sake. /cc @addaleax

Reply:
One option would be to use something like 'PlatformWorkerThread' here.